ant.go

```diff
@@ -23,7 +23,7 @@ const (
 )
 
 type Ant struct {
-	port    uint16
+	port    int
```

Comment:
I actually preferred the uint16 but this gradually bubbled through the code when I added the CLI flag mapping which is an int.